tasks/push-to-cdn/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Add mocks to the beginning of task step script
+TASK_PATH="$1"
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+yq -i '.spec.steps[0].script = load_str("'$SCRIPT_DIR'/mocks.sh") + .spec.steps[0].script' "$TASK_PATH"
+
+# Create a dummy exodus secret (and delete it first if it exists)
```

Comment:
Can you adjust the secret name to be something more related to the task name? If we have multiple tasks with exodus secrets, we can run into issues here when running CI